apm-agent-core/src/main/java/co/elastic/apm/agent/configuration/SpanConfiguration.java

```diff
@@ -57,6 +57,16 @@
         .dynamic(true)
         .buildWithDefault(TimeDuration.of("5ms"));
 
+    private final ConfigurationOption<TimeDuration> exitSpanMinDuration = TimeDurationValueConverter.fineDurationOption("ms")
```

Comment:
We're trying to get rid of default units. The reason they still exit is backward compatibility. For new options, please don't specify a default duration suffix.
Unfortunately, this is not really reflected in `TimeDurationValueConverter`. I think we should change `withDefaultFineDuration` and `fineDurationOption` so that they don't accept a default duration and provide an empty string to the `TimeDurationValueConverter` constructor. `TimeDurationValueConverter#withDefaultDuration(String)` and `durationOption(String) should be deprecated and we should create overloads that don't accept a default duration.

Reply:
Ok, should I do this in this PR or should it be a separate PR?

Reply:
@felixbarny #2503